Docs/configuration.md

```diff
@@ -1,10 +1,6 @@
-Imgbot supports optional configuration through a `.imgbotconfig` json file.
-This is not a required step to using Imgbot and is only for more advanced scenarios.
+Optionally, for more advanced scenarios, you can configure Imgbot using an `.imgbotconfig` json file in the root of your GitHub repository <!-- exact filename? can we remove the json? --> and set to your liking.
 
-This file should be placed in the root of the repository and set to your liking.
-See [this past issue](https://github.com/dabutvin/Imgbot/issues/49) for details about the location for this file.
```

Comment:
I checked the issue and could not find out how it helps users to understand why or why not the file is placed in the root folder.